openstudiocore/src/utilities/geometry/PointLatLon.cpp

```diff
@@ -20,9 +20,9 @@
 #include "PointLatLon.hpp"
 #include "Point3d.hpp"
 
-#include "../../GeographicLib/Geodesic.hpp"
-#include "../../GeographicLib/LocalCartesian.hpp"
-#include "../../GeographicLib/UTMUPS.hpp"
+#include <GeographicLib/Geodesic.hpp>
```

Comment:
@axelstudios GeographicLib is built and installed in the build directory, I'm not sure how this was working anywhere

@kbenne FYI